admin/importitems.php

```diff
@@ -89,7 +89,7 @@ function additem($itemtoadd,$item,$questions,$qset) {
 				$tosets .= ','.$set;
 				//DB $valsets .= ',\''.$item[$itemtoadd][$set].'\'';
 				$valsets .= ',:'.$set;
-				$qarr[':'.$set] = $item[$itemtoadd][$set];
+				$qarr[':'.$set] = Sanitize::stripHtmlTags($item[$itemtoadd][$set]);

```

Comment:
Some of these items contain HTML and should not be stripped of tags.  Items that might contain raw HTML are sanitized in lines earlier.